lib/Controller/BookingController.php

```diff
@@ -137,10 +149,12 @@ public function bookSlot(int $appointmentConfigId,
 			return JsonResponse::fail(null, Http::STATUS_NOT_FOUND);
 		}
 
-		// validate slot is available
-		// write slot, preparation duration and follow up duration events
-//		$this->bookingService->book($startTimeInTz, $endTimeInTz, $config, $start);
+		try {
+			$slot = $this->bookingService->book($config, $startTimeInTz, $endTimeInTz, $start, $name, $email, $description);
+		} catch (ServiceException $e) {
```

Comment:
also catch client exceptions and respond accordingly. and I don't think any service exception should be answered with a 404 :)